islandora-connector-broadcast/src/main/cfg/ca.islandora.alpaca.connector.broadcast.cfg

```diff
@@ -0,0 +1,7 @@
+# Incoming queue/topic on the broker of your choice.
+# Replace the 'broker' prefix with the broker for your system.
+# E.g. activemq, zeromq, etc... 
+input.stream=broker:queue:islandora-connector-broadcast
+
+# Header that contains the recipient list
+recipients.header=ca.islandora.alpaca.connector.broadcast.recipients
```

Comment:
This is just my own personal preference, but I like to use a convention more like this for naming headers: `CamelClawBroadcasterRecipients`. This makes it a little easier to refer to the header in `simple` expressions, since you can refer to it like this: `simple("${header.CamelClawBroadcasterRecipients}")`. There certainly are camel components that use the `ca.islandora.alpaca....` convention, and Fedora's JMS module uses `org.fcrepo.jms.___`, but for most things that run entirely inside Camel, I've seen the pattern I recommended above.